src/components/RunButton.js

```diff
@@ -20,14 +20,35 @@ const useStyles = makeStyles((theme) => ({
   }
 }));
 
+function replacer(key, value) {
+  if (key === 'config') {
+    return undefined;
+  }
+  return value;
+}
+
 export default function RunButton(props) {
   const classes = useStyles();
 
   //Sets the shouldRunSUSHI to true
   async function handleClick() {
-    props.onClick(true);
     const outPackage = await runSUSHI(props.text);
-    console.log(outPackage);
+    let jsonOutput = JSON.stringify(outPackage, replacer, '\t');
```

Comment:
If you do end up using the `<pre>` tag, the tabs look pretty indented. I often use 2 spaces to indent JSON, so you can replace the `'\t'` here with `2`.